device/core/test/blob_upload/_file_upload_api_test.js

```diff
@@ -45,9 +45,9 @@ describe('FileUploadApi', function() {
       };
       var testTransport = new Transport();
       var fileUpload = new FileUploadApi(fakeAuthenticationProvider, testTransport);
-      fileUpload.getBlobSharedAccessSignature('blobName', 'sas', function() {});
+      fileUpload.getBlobSharedAccessSignature('blobName', function() {});
       assert.equal(callCount, 1);
-      fileUpload.notifyUploadComplete('correlationId', 'sas', { isSuccess: true, statusCode: 200, statusDescription: 'test' }, function() {});
```

Comment:
> 'sas',  [](start = 54, length = 8)

What is going on with the loss of these arguments?

Reply:
The method doesn't have that parameter:

`notifyUploadComplete(correlationId: string, uploadResult: BlobUploadResult, done?: (err?: Error) => void): void {`

[source](https://github.com/Azure/azure-iot-sdk-node/blob/734bd35f0a48b519ee4de233d1762e7ef2d4521c/device/core/src/blob_upload/file_upload_api.ts#L95)

Reply:
Won't fix.

---
In reply to: [222780291](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222780291) [](ancestors = 222780291)